focus-android/app/src/main/AndroidManifest.xml

```diff
@@ -100,7 +100,7 @@
         <activity android:name=".activity.MainActivity"
             android:launchMode="singleTask"
             android:exported="true"
-            android:windowSoftInputMode="adjustResize|stateAlwaysHidden"
+            android:windowSoftInputMode="adjustResize"
```

Comment:
I wonder why we diverge from [Fenix](https://github.com/mozilla-mobile/firefox-android/blob/eca2ac694b075bd0697fad2139652b01502cdd14/fenix/app/src/main/AndroidManifest.xml#L117)

Reply:
That part of the Focus code is from March 2018 before Fenix existed. There were different flows at that time in Focus so I am not sure what was the original purpose.